### PR TITLE
LPD-19468 Release FF for AI Create Images in DM 

### PR DESCRIPTION
**Pull request by ambrinchaudhary**

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -7271,6 +7271,8 @@ feature.flag.COMMERCE-13024.description=Users can view and edit custom object fi
 feature.flag.COMMERCE-13024.title=Organization Management Widget and Custom Object Fields
 feature.flag.LPD-6368.description=Users can scope search result rankings (i.e., everything, site, or blueprint). Rankings only apply to searches with a matching scope. When enabled, existing rankings apply to searches with the everything scope. When disabled, result rankings with the site or blueprint scope disappear.
 feature.flag.LPD-6368.title=Scoping Result Rankings
+feature.flag.LPD-10793.description=Generate and save AI images in the Documents & Media application using OpenAI.
+feature.flag.LPD-10793.title=AI image generation in DAM
```
Comment (marcogalluzzi) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:7275]: Titles should be capitalized: _AI Image Generation in DAM_

Comment (ambrinchaudhary) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: Updated, thank you!

**Comment by ambrinchaudhary** [comment]
Although there is one task pending in the epic (https://liferay.atlassian.net/browse/LPD-15186) I think it is safe to send this change.

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:stop

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
Failures not related to this pull.

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Pull request closed without merging**